### DISCUSSION TITLE
Add track after currently playing track when in Dynamic mode

### DISCUSSION THREAD

**Discussion by farid-fari** [strawberrymusicplayer/strawberry#1011] [General]
When playing in dynamic shuffle mode, and I double click on a track in the explorer, I don't want it added after the arbitrary number of random tracks currently there -- I want it played after the currently playing track, i.e. as the next track.

**Comment by jonaski** [comment]
GitHub issues isn't user support.

**Reply by farid-fari** [reply]
I'm not asking for support, this is a feature/enhancement request

**Reply by jonaski** [reply]
I don't take enhancement/feature requests at this time, it's written in the issue template and the README file for this project.
